uisdk/src/main/java/com/karhoo/uisdk/screen/booking/address/addressbar/AddressBarPresenter.kt

```diff
@@ -136,8 +136,8 @@ internal class AddressBarPresenter(view: AddressBarMVP.View,
         }
     }
 
-    override fun subscribeToBookingStatus(bookingStatusStateViewModel: BookingStatusStateViewModel): Observer<BookingInfo> {
-        setCurrentBookingStatus(bookingStatusStateViewModel)
+    override fun subscribeToBookingStatus(journeyDetailsStateViewModel: JourneyDetailsStateViewModel): Observer<JourneyDetails> {
```

Comment:
This method name needs refactoring to `subscribeToJourneyDetails`. It may become confusing in the future otherwise since `BookingStatus` is no longer referred to as such.